playground/package.json

```diff
@@ -18,6 +20,7 @@
   "scripts": {
     "build": "npx tailwindcss -i ./src/styles.css -o ./public/styles.css --minify",
     "watch": "npx tailwindcss -i ./src/styles.css -o ./public/styles.css --watch",
-    "postinstall": "./postinstall.sh"
+    "postinstall": "./postinstall.sh",
+    "worker-bundle": "rspack --config ./rs.config.js"
```

Comment:
I suppose we can have

```
tailwind:watch
tailwind:build
worker:build
```

and then `build` calls `worker:build && tailwind:build`